guides/installation/src/git/github.md

```diff
@@ -0,0 +1,228 @@
+# Using Git with GitHub
+GitHub is a cloud-based Git repository hosting service we use to host all of our software.
+Our repositories can be found [here](https://github.com/uavaustin).
+
+## Contributing to a Repository
+**Starting a Local Repository**
+
+Go to the repository link of the project that you want to work on or create your own. 
+
+Copy the remote URL onto your clipboard:
+<img src="../img/remotes-url.png" alt="remote-url" width="600">
+
+On your local machine and desired directory:
+```
+$ git clone <remote-url>
+```
+If you are making your own repository:
+```
+$ git init
+$ git remote add origin <remote-url>
+```
+**Updating a Local Repository**
+
+Sometimes there will be updates to master in the remote repository; to merge/inspect
+these changes, consider the following commands.
+```
+$ git fetch origin
+```
+Fetch will download data from the remote repository, but will not modify anything locally.
+This is a good way to check up on the progress of other collaborators.
+```
+$ git pull origin master
+```
+Pull runs the aforementioned fetch and then merges the changes in the remote repository to your
+local environment. Pull is equivalent to a "fetch" and then a "merge". As such, there is a chance for
+merge conflicts if you share modified files or lines with the remote repository.
+
+**Updating a Remote Repository**
+
+When working on a feature, it's a good idea to create your own branch instead of working directly on master.
+To update the remote repository to show your new branch:
+```
+$ git push -u origin <branch-name> # -u flag needed if branch not present on remote
+```
+You can see if the push was successful by checking your pushed branch on GitHub by clicking on the branch
+drop-down underneath "Code":
+
+<img src="../img/git-branches-2.png" alt="git-branches" width="400">
+
+## An Example
+We'll do an example of all this real fast.
+
+*Note that this example is using the command-line Git client, however you may
+also do this with something like
+[GitHub&nbsp;Desktop](http://desktop.github.com) if you wish.*
+
+### Creating Feature Branches
+
+Say you have a task telling you to add a stock photo of a cat to a GUI. In this
+case, we'll add a Scottish Fold since they're by far one of the cutest cats
+there are.
+
+
+<img src="https://i.imgur.com/UklMnBr.jpg" alt="Cat Picture" style="width:30%">
+
+We're going to want to make a new branch off of *master* to work on, so we
+don't bother someone else working on the same project.
+
+Feature branches should have a short, descriptive name. Our naming convention
+for feature branches is to simply use lower-case names separated by hyphens.
```

Comment:
I'd prefer to see here something like "Branch naming scheme will depend on the team lead, but here we will simply ...". 
I prefer `initials-feature` to know who is working on what.

Reply:
Fixed